### ISSUE TITLE
hexo error

### ISSUE THREAD

**Issue by TristanWright** [Kitware/paraviewweb#55]
From fresh, naive installation: 

```
=> Serve documentation:


> paraviewweb-site@0.0.0 server /Users/tristan/Documents/work/paraviewweb/documentation/www
> hexo server

ERROR Local hexo not found in ~/Documents/work/paraviewweb/documentation/www
ERROR Try running: 'npm install hexo --save'

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/5.4.1/bin/node" "/usr/local/bin/npm" "run" "server"
npm ERR! node v5.4.1
npm ERR! npm  v3.5.3
npm ERR! code ELIFECYCLE
npm ERR! paraviewweb-site@0.0.0 server: `hexo server`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the paraviewweb-site@0.0.0 server script 'hexo server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the paraviewweb-site package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     hexo server
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs paraviewweb-site
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls paraviewweb-site
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/tristan/Documents/work/paraviewweb/documentation/www/npm-debug.log
```

**Comment by TristanWright** [comment]
Still occurs after `brew install homebrew/science/vips` for "sharp."

**Comment by TristanWright** [comment]
required vips `8.2.2`, had `8.2.1` 😑

`brew update && brew upgrade` fixed that bit, still failing.

**Comment by TristanWright** [comment]
closed with #56
